admin/js/swonpage-navigation.js

```diff
@@ -173,24 +117,55 @@
             var list = arguments[0];
             var elements = [];
             elements.push('<b style="display:block;padding:5px;padding-bottom:10px;padding-top:30px">Choose page layout to use:</b>');
+
             for (var i = 0; i < list.length; i++) {
                 var isPreview = list[i].path.indexOf('.jpg') > 0 || list[i].path.indexOf('.jpeg') > 0 || list[i].path.indexOf('.png') > 0 || list[i].path.indexOf('.gif') > 0;
```

Comment:
If `isPreview` is important for code understanding, then maybe we should do:
`var isPreview = isImagePath(list[i].path);`